elytron/src/main/resources/schema/wildfly-elytron_7_0.xsd

```diff
@@ -748,6 +748,20 @@
                         </xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:attribute name="syslog-format" type="xs:string">
```

Comment:
Since the schema has been bumped, these changes should be in ```wildfly-elytron_8_0.xsd```.

Reply:
Fixed